### ISSUE TITLE
Add FQDN output to digitalocean_record resource types

### ISSUE THREAD

**Issue by kyl191** [hashicorp/terraform#5064]
Hi,
I'm creating a DNS record with `digitalocean_record`, and want to be able to use the resulting FQDN. 

`Cloudflare_record` exports the `hostname` attribute, which is the FQDN, could a similar thing be added for `digitalocean_record`?

**Comment by stack72** [comment]
Hi @kyl191,

this isn't available as part of the SDK - the SDK has the following fields in the struct:

```
type DomainRecord struct {
	ID       int    `json:"id,float64,omitempty"`
	Type     string `json:"type,omitempty"`
	Name     string `json:"name,omitempty"`
	Data     string `json:"data,omitempty"`
	Priority int    `json:"priority,omitempty"`
	Port     int    `json:"port,omitempty"`
	Weight   int    `json:"weight,omitempty"`
}
```

I *think* we would have to ask for a change to be made to their API as it doesn't expose it either AFAICT

https://developers.digitalocean.com/documentation/v2/#domain-records

Paul

**Comment by kyl191** [comment]
Hi Paul, 
DIgital Ocean might not expose it, but I think it would be possible to do it in terraform itself - we know both parts of the FQDN.

I've created a workaround for my usecase with a module (https://github.com/kyl191/terraform-cloudflare-modules/blob/master/create-fqdn/main.tf) - essentially pass the domain + subdomain, and get an output of the two combined.

Thanks,
Kyle

**Comment by radeksimko** [comment]
Agreed with @kyl191 
we do this for Route53 records already:
https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_route53_record.go#L308-L310

**Comment by stack72** [comment]
thanks @radeksimko - will add it
@kyl191 I will open a PR for this shortly

**Comment by stack72** [comment]
Hi @kyl191 - this has been merged and will appear in 0.6.12 :)
